internal/testutils/networking/iptables.go

```diff
@@ -44,3 +45,8 @@ func SetFilterForwardPolicies(t *testing.T, policy string) {
 		})
 	}
 }
+
+func FirewalldRunning() bool {
+	state, err := exec.Command("firewall-cmd", "--state").CombinedOutput()
```

Comment:
As this is brand new; perhaps we could pass the context?

```suggestion
func FirewalldRunning(ctx context.Context) bool {
	state, err := exec.CommandContext(cox, "firewall-cmd", "--state").CombinedOutput()
```